manifests/install.pp

```diff
@@ -4,16 +4,19 @@
 #
 class consul::install {
 
-  if $::consul::data_dir {
-    file { $::consul::data_dir:
+  if ($::consul::data_dir) and ($::consul::install_method != 'docker') {
```

Comment:
Why is there no data_dir? I would prefer to have a permanently mounted volume outside the container. That way if I have to nuke docker for some reason, puppet can reproduce the image and run and all it has to do is pickup the data on disk